src/components/Footer/index.tsx

```diff
@@ -1,7 +1,45 @@
-const Footer = () => {
+'use client';
+import { useMemo } from "react";
+
+/**
+ * Footer component displaying contact details and a copyright section.
+ * This component includes an email, phone number, and location with corresponding icons.
+ *
+ * @returns {JSX.Element} - The rendered footer component.
+ *
+ * @author Lishin V S
+ */
+
+const Footer = ():JSX.Element =>{
+
+    const contactDetails = useMemo(() => [
+        { label: "email", value: "ssladmin@nitc.ac.in", icon: "/Footer/mail.svg" },
+        { label: "phone", value: "0495 228 6864", icon: "/Footer/phone.svg" },
+        { label: "location", value: "ITL302 - 2nd Floor", icon: "/Footer/location.svg" },
+    ], []);
+    
     return (
-        <footer>
+        <footer className="px-8 bg-[#202127] border-t border-white md:px-10">
+            <div className="font-bold text-sm flex flex-wrap justify-center  items-center py-5 sm:justify-between md:text-base">
```

Comment:
For screens >= lg, everything should be in 1 row, reduce font size if required
for all other screens, everything should be in 1 column
(including all contact details and copyrights)

Reply:
I have made the changes. Let me know if there is anything else to change